include/fiction/technology/charge_distribution_surface.hpp

```diff
@@ -97,6 +97,15 @@ class charge_distribution_surface<Lyt, false> : public Lyt
         }
         return sidb_charge_state::NONE;
     }
+    /**
+     * Returns the number of SiDBs assigned to a charge state
+     *
+     * @return number (uint64_t)
```

Comment:
Number of assigned charge states.

No need to mention the data type. It is given by the function definition.